### DISCUSSION TITLE
How to add a page without including it in the sidebar or navbar?

### DISCUSSION THREAD

**Discussion by jmhimara** [quarto-dev/quarto-cli#7312] [Q&A]
### Description

In a website project, I want to create a page that's not in the sidebar or navbar, just a page on its own that can be linked to by another page. 

Should be easy, but I can't seem to find anything in the docs about it.

**Comment by mcanouil** [comment]
There is nothing to do.
If there is a Quarto document that does not start with `_` and if you did not exclude that file in the render targets, the document will be rendered and the page will be created.

If you are wondering about linking this is documented under "linking", see <https://quarto.org/docs/websites/index.html#linking>.

**Reply by jmhimara** [reply]
It doesn't seem to be working. When I link, it just links to the raw qmd file and does not produce an html (qmd file is copied in the output folder).

**Reply by mcanouil** [reply]
Please provide a reproducible example of your case.

**Reply by jmhimara** [reply]
Nevermind, I think I figured it out. You have to add the `render` option underneath your project in `_quarto.yml`. 

```qmd
project:
  render: 
    - index.qmd
    - about.qmd
    - new_file.qmd
  type: website
```

Otherwise it doesn't pick up the new file. The downside to this is that now you have to include all the qmd files again (the ones already included inside the navbar) but it's not a big deal.

**Reply by cscheid** [reply]
Without an explicit `render` list, all .qmd files that don't start with `_` should be rendered whether or not they're linked to anywhere else in the document. If you can create a minimal example that triggers the problem, then please let us know because that would be a quarto bug.

**Reply by jmhimara** [reply]
Ok strange, now it is working correctly. Yesterday, the `new_file.qmd` wasn't being rendered by quarto. Perhaps something to do with the Vscode preview rather than quarto itself. 

Anyway, all good now.

**Comment by s2t2** [comment]
FYI I am seeing the behavior referenced here as well
https://github.com/quarto-dev/quarto-cli/discussions/7312#discussioncomment-7345228

The files don't render by default (book project). They also don't render when using the `render` option. I'm a bit stumped.

Here you will see certain files included in the render section, however they are actually not rendered:

<img width="1291" alt="Screenshot 2024-08-15 at 1 53 44 PM" src="https://github.com/user-attachments/assets/b1cf2df7-22ae-471e-868c-962671c37abb">

Also links to these files appear as raw markdown:

<img width="1067" alt="Screenshot 2024-08-15 at 1 54 50 PM" src="https://github.com/user-attachments/assets/80dadb74-7ca5-48ca-9d16-c1a0b844dc8c">


https://github.com/quarto-dev/quarto-cli/discussions/10531
